### ISSUE TITLE
ModuleNotFoundError: No module named 'ffmpeg_dl'

### ISSUE THREAD

**Issue by Suika** [AlexAplin/nndownload#157]
Running on `Python 3.6.15 (default, Sep 23 2021, 15:41:43) [GCC] on linux`
PyPi package of nndownload==1.16.1 exhibits the following issue:
```
raceback (most recent call last):
  File "/archive-venv/bin/nndownload.py", line 33, in <module>
    from ffmpeg_dl import FfmpegDL, FfmpegDLException
ModuleNotFoundError: No module named 'ffmpeg_dl'
```
Also pointed out in the last two comments of the following issue: #153

**Comment by Suika** [comment]
Naturally it will not work, because the `ffmpeg_dl` module is in `/archive-venv/lib/python3.6/site-packages/nndownload` and `/archive-venv/bin/nndownload.py` is not a link to `/archive-venv/lib/python3.6/site-packages/nndownload/nndownload.py`  but a fully fledged script.

Since the call happens in `/archive-venv/bin/`, there is nothing to import from.

**Comment by fireattack** [comment]
https://github.com/AlexAplin/nndownload/pull/156 should fix it

**Comment by AlexAplin** [comment]
This should be resolved with v1.16.2
